modules/apps/expando/expando-taglib/src/main/resources/META-INF/resources/custom_attribute/page.jsp

```diff
@@ -98,12 +94,6 @@ ExpandoBridge expandoBridge = ExpandoBridgeFactoryUtil.getExpandoBridge(company.
 								if (value != null) {
 									valueDate.setTime((Date)value);
 								}
-								else if (defaultValue != null) {
-									valueDate.setTime((Date)defaultValue);
-								}
-								else {
-									valueDate.setTime(new Date());
-								}
```

Comment:
@IstvanD, please add the below code back for Date type. It will keep consistent with the current pr did.

```
else {
	valueDate.setTime(new Date());
}
```

I notice Date type custom field always existed default value(refer to ExpandoValueImpl#getDate()). That is to say, `value` always is not null. Even though we use api set defaultValue is null for Date type custom field, when getting the value, it is still not null. This may be another issue. I think we can skip it.

After you change it, you may just git push force the change.

Thanks,
Hai